sharktank/sharktank/types/theta.py

```diff
@@ -537,11 +537,11 @@ def _dataset_load_helper(
 ) -> Dataset:
     path = Path(path)
     suffixes = path.suffixes
-    if file_type == "gguf" or suffixes == [".gguf"]:
+    if file_type == "gguf" or suffixes[-1] == ".gguf":
         from . import gguf_interop
 
         return gguf_interop.load_file(path)
-    elif file_type == "irpa" or suffixes == [".irpa"]:
+    elif file_type == "irpa" or suffixes[-1] == ".irpa":
```

Comment:
Oh, I ran into this suffix issue on my own and sent a PR: https://github.com/nod-ai/sharktank/pull/15

Can we get some of these merged?

Reply:
Please do. PRs are piling up. It's there an access issue we need to fix?